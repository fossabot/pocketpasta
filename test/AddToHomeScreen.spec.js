import { shallowMount } from '@vue/test-utils';
import component from '@/components/AddToHomeScreen.vue';

describe('AddToHomeScreen', () => {
  const factory = () => shallowMount(component, {});

  test('mounts properly', () => {
    const wrapper = factory();
    expect(wrapper.isVueInstance()).toBeTruthy();
  });

  test('renders properly', () => {
    const wrapper = factory();
    expect(wrapper.html()).toMatchSnapshot();
  });

  test('events', () => {
    const wrapper = factory();
    wrapper.trigger('beforeinstallprompt');
    wrapper.trigger('click');
  });
});
