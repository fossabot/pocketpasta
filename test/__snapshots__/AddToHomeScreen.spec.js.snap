// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddToHomeScreen renders properly 1`] = `"<v-btn icon=\\"\\" aria-label=\\"get app\\" class=\\"d-none\\"><v-icon>get_app</v-icon></v-btn>"`;
